environment/simulation/simulation.py

```diff
@@ -166,7 +167,7 @@ def _process_events(self):
             elif event.type == KEYDOWN and event.key == K_p:
                 pygame.image.save(self._screen, "swarm_ball_simulation.png")
 
-    def redraw(self):
+    def _update_screen(self):
         self._screen.fill(THECOLORS["white"])
         if self.debug is True:
```

Comment:
Wait... czy ten if nie sprawia czasem, że z opcją debug nie wyświetlają nam się klasty, przeciwnik itp? (wiem, że nic tu nie zmieniałeś, ale teraz mi to w oko wpadło)

Reply:
Uu racja racja

Reply:
dodałem commit, może tak będzie sensowniej

Reply:
To znaczy twój komentarz zwrócił mi uwagę na tak probem, że po pierwsze niby nie do końca dzieje się to co mówisz bo debug wyświetla absolutnie wszystko co jest w symulacji, nawet za dużo, ale własnie w obecnym kodzie gdyby ktoś przypadkiem zostawił sieci włączony debug to zrobiłby się straszny bałagan. Więc zrobiłem tak żeby opcja debugu byla tylko i wylacznie na ekran monitora